src/main/java/testUI/AndroidUtils/ADBUtils.java

```diff
@@ -31,10 +31,10 @@ public class ADBUtils {
     private static String emulatorFolder = "/emulator/";
     public static String MAC_CHROME_DRIVER =
             "/usr/local/lib/node_modules/appium/node_modules/appium-chromedriver/chromedriver/mac" +
-                    "/chromedriver";
+                    "/chromedriver*";
```

Comment:
Does this actually work? 

I created `random_123.txt` file and tried calling `File.exists("random_*")` and it returned `false`

Reply:
The method file doesn't admit wildcards, it's not the same thing, we are using that as a command line in a bash terminal, and that admits it. you can try it out like that.